models/notification.go

```diff
@@ -238,6 +239,42 @@ func notificationsForUser(e Engine, user *User, statuses []NotificationStatus, p
 	return
 }
 
+// APIFormat converts a Notification to api.NotificationThread
+func (n *Notification) APIFormat() *api.NotificationThread {
```

Comment:
I would like to move all the convertion to `route/conversion` package

Reply:
@lunny should I creat a refactor-PR for all other APIFormat() func?

Reply:
cant find `route/conversion`